modules/tinymce/src/core/main/ts/fmt/FormatUtils.ts

```diff
@@ -198,6 +198,15 @@ const areSimilarFormats = (editor: Editor, formatName: string, otherFormatName:
   });
 };
 
+const isSelectorFormat = (format: Format): format is SelectorFormat =>
+  Obj.hasNonNullableKey(format as any, 'selector');
+
+const isBlockFormat = (format: Format): format is BlockFormat =>
+  Obj.hasNonNullableKey(format as any, 'block');
+
+const isInlineFormat = (format: Format): format is InlineFormat =>
+  Obj.hasNonNullableKey(format as any, 'block');
```

Comment:
I believe this should be `'inline'` instead of `'block'`

Reply:
Yeah it should be and is why the tests are failing. Sorry, I should of mentioned I needed to fix that.

Reply:
Fixed